src/eterna/mode/PoseEdit/PoseEditMode.ts

```diff
@@ -1719,11 +1845,24 @@ export default class PoseEditMode extends GameMode {
         }
         this._poses[poseIndex].forcedHighlights = this.getForcedHighlights(targetIndex);
 
-        if (this._puzzle.nodeID === 2390140) {
+        if (this._puzzle.nodeID === PuzzleID.TheophyllineRibozymeSwitch) {
```

Comment:
This is an artifact of the old AuxInfo infrastructure that I updated to conform with the annotations infrastructure. I had spoken to @luxaritas and we agreed that would be the best course of action. This is a special case that seems to have been hard-coded. Otherwise nothing new I've implemented should occur at this level.

Reply:
Right. Do you think the annotations scheme will make it easier to get rid of this hardcoding in a subsequent iteration? It seemed perhaps a little necessary then because I don't know if there was widespread read-in of aux info from the TargetConditions json blob. Now that that's standard with annotations, it seems feasible to fix this in the future, even if not in this PR.

Reply:
It should be easily resolved in a future yes